.evergreen/config_generator/components/c_std_compile.py

```diff
@@ -39,9 +39,10 @@
     ('rhel94',     'gcc',    None, [99, 11, 17, 23]), # GCC 11.4 (max: C2x)
     ('rhel95',     'gcc',    None, [99, 11, 17, 23]), # GCC 11.5 (max: C2x)
 
-    ('windows-vsCurrent', 'vs2017x64', None, [99, 11, 17, 'latest']), # Max: C17, clatest (C2x)
-    ('windows-vsCurrent', 'vs2019x64', None, [99, 11, 17, 'latest']), # Max: C17, clatest (C2x)
-    ('windows-vsCurrent', 'vs2022x64', None, [99, 11, 17, 'latest']), # Max: C17, clatest (C2x)
+    ('windows-vsCurrent', 'vs2015x64', None, [99, 11,   ]), # Max: C11
+    ('windows-vsCurrent', 'vs2017x64', None, [99, 11, 17]), # Max: C17
```

Comment:
> C11 and C17 tasks for VS 2017 and newer are currently failing

This may have been fixed. [std-c11-windows-2019-vs2017-x64-compile](https://spruce.mongodb.com/task/mongo_c_driver_std_matrix_std_c11_windows_2019_vs2017_x64_compile_patch_fee885a3a4a994175dc29b7f6aee085a634ef280_67f83f0884dfad0007877f6f_25_04_10_21_58_35/logs?execution=0) passed. [std-c17-windows-2019-vs2017-x64-compile](https://spruce.mongodb.com/task/mongo_c_driver_std_matrix_std_c17_windows_2019_vs2017_x64_compile_patch_fee885a3a4a994175dc29b7f6aee085a634ef280_67f83f0884dfad0007877f6f_25_04_10_21_58_35/logs?execution=0) failed, but appears to be due to not supporting C17:

> ```
> Latest C standard 11 is older than 17
> ```

```suggestion
    ('windows-vsCurrent', 'vs2017x64', None, [99, 11    ]), # Max: C11
```

Reply:
```
Selecting Windows SDK version 10.0.19041.0 to target Windows 10.0.20348.0.
```
It looks like it managed to select the patched SDK version as the target. I'll admit I don't fully understand what this means (I was expecting "Selecting Windows SDK version 10.0.20348.0" insetad). 🤔 However, no complaints if it works.